packages/shared/src/components/post/PostNavigation.tsx

```diff
@@ -50,7 +50,7 @@ function PostNavigation({
       {onPreviousPost && (
         <SimpleTooltip content="Previous">
           <Button
-            className="-rotate-90 btn-secondary"
+            className="border-none -rotate-90 btn-secondary"
```

Comment:
```suggestion
            className="-rotate-90 btn-tertiary"
```

I think this should do looking at the design spec.

Reply:
Ah sorry about that. Updated. I would expect `Ter` to be up here also, but it is not. 🤦 

<img width="315" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/b56f21cc-a70f-409e-a3c6-b32b91f79499">